src/components/WorkOrders/JobRow.js

```diff
@@ -0,0 +1,50 @@
+import PropTypes from 'prop-types'
+
+const dateToStr = (date) =>
+  date.toLocaleString('en-GB', {
+    year: 'numeric',
+    month: 'long',
+    day: 'numeric',
+  })
+
+const hourTosTr = (date) =>
```

Comment:
This const name is a bit confusing as it suggests converting hours to a string but i think you are converting a date right?